docs/getting-started/installing-regression-games.mdx

```diff
@@ -4,6 +4,115 @@ sidebar_label: 'Installing Regression Games'
+
+```
+https://github.com/Regression-Games/RGUnityBots?path=src/gg.regression.unity.bots.ecs#v0.0.27
+```
+
+## Authenticate through the Regression Games SDK
+
+Once our core SDK and applicable extensions have been added to your project, close the Package Manager.
+You will be greeted by a prompt for Regression Games login details.
+
+Create a personal account through [the Regression Games website](https://play.regression.gg/signup).
+This step is simple, and will give you access to many
+features such as saving [Gameplay Sessions](../core-concepts/gameplay-sessions/gameplay-sessions-getting-started)
+and creating powerful [Validation Suites](../core-concepts/validation-suites/validation-suites-getting-started).
+
+:::caution
+You can submit your new account credentials through the Unity prompt now,
+but this method is less secure than creating and using an API key.
+We recommend taking an extra moment to follow the section below.
+:::
+
+### Set an API Key
+
+Generate a new API key through your account settings in the Regression Games client, and copy the generated value to your clipboard.
+
+![Generate a new API key](../core-concepts/img/api-keys/create-new-api-key.gif)
+
+:::info
+To learn more about our API keys, see the [Authenticating with API Keys](../core-concepts/authenticating-with-api-keys) section.
+:::
+
+Do not submit any information to the welcome prompt in Unity.
+Instead, close the prompt and navigate to `Edit` > `Project Settings` > `Regression Games`.
+Paste the key you just generated into the `RG API Key` field, then close the project settings.
+
+![Set an API Key in Unity](../core-concepts/img/api-keys/set-api-key-unity.png)
+
+## Restart Unity
+
+Now is a good time to restart Unity and any code editor you may have open (e.g. Rider or Visual Studio).
+Without this step, Unity and your code editor may not properly resolve your new packages even after Unity recompiles the scripts.
+
+## Add the Regression Games Overlay to Your Scene
+
+Our core package provides an in-game overlay that acts as a UI for features such as performing
+[input recording and playback](../core-concepts/gameplay-sessions/gameplay-sessions-getting-started)
+and constructing and running [Bot Sequences](../core-concepts/bot-sequences/getting-started-with-bot-sequences-and-segments),
+but it also drives our data collection behind-the-scenes.
+This means the overlay prefab **must** be present in each scene where you want to use any Regression Games features.
+However, you can hide the visibility of the overlay through the project settings under
+`Edit` > `Project Settings` > `Regression Games` if needed.
+
+Find the `RGOverlayCanvas` prefab by navigating to
+`Packages` > `Regression Games Unity Bots` > `Runtime` > `Prefabs` and dragging it into your scene's object hierarchy.
+Make sure that the canvas is at the top of the hierarchy, so that it is always visible and interactable.
+Repeat this for each scene that must have access to Regression Games features.
```

Comment:
Can we get away with adding it once and setting it 'don't destroy on load'?

Reply:
We can either leave this as is, or write a paragraph explaining to them that it needs to be in your 'first' scene in all situations.  Telling them every scene was 'safer' as it is not uncommon to use one scene as the first scene in the IDE but a different one in a built runtime.

Reply:
I'm leaning towards the "safer" route. If we receive negative feedback on it then we can clarify